test/end_to_end/recovery_SUITE.erl

```diff
@@ -68,6 +68,7 @@ replace_everything(_Config) ->
         testutil:put_altered_indexed_objects(Book1, BKT, KSpcL1),
     ok = testutil:check_indexed_objects(Book1, BKT, KSpcL2, V2),
     compact_and_wait(Book1, 1000),
+    compact_and_wait(Book1, 1000),
```

Comment:
Surprising, need to wait one more second now?

Reply:
The test was that after one loop there is no further compaction work to be done.  However, there is some potential randomness here, and so sometimes there may have still been viable compaction work for a second loop.  so now we test all compaction work is done in two attempts, not just one.